add.js

```diff
@@ -0,0 +1,3 @@
+function add(a,b){
```

Comment:
- Provide Jsoc documentation for better understanding.

Reply:
resolved